dev-docs/bidders/adform.md

```diff
@@ -9,7 +9,7 @@ usp_supported: true
 tcf2_supported: true
 prebid_member: true
 pbjs: true
-pbs: false
+pbs: true
 userIds: britepoolId, criteo, id5Id, identityLink, liveIntentId, netId, parrableId, pubCommonId, sharedId, unifiedId
 gvl_id: 50
```

Comment:
Line 20 
```
{: .table .table-bordered .table-striped }
```
seems to not getting displayed properly in github: https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/adformPbjs.md

Reply:
For me, it looks the same way as in other adapters docs.  Maybe you could tell me what exactly should be changed?

Reply:
This will be fine when it shows up in the processed Jekyll. I confirmed on my local sever.